README.md

```diff
@@ -88,6 +88,48 @@ The easiest way to run release please is as a GitHub action:
   convention, [release-please](https://github.com/googleapis/release-please)
   will start creating Release PRs for you.
 
+#### Automating publication to npm
+
+With a few additions, the Release Please action can be made to publish to
+npm when a Release PR is merged:
+
+```yaml
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bcoe/release-please-action@v1.2.1
```

Comment:
Can we use `GoogleCloudPlatform` now?

Reply:
I was figuring I'd land everything against `bcoe/` then update to GoogleCloudPlatform once I've done a first release from that location.